docs/data/routes/docs/nextjs/deploying-to-production/vercel/en.md

```diff
@@ -12,17 +12,24 @@ This guide will demonstrate deploying the Next.js sample app that's generated by
 1. Execute the steps provided in the Next.js  guide for [Getting Started](https://nextjs.org/docs/deployment#getting-started) with deployment.
 2. Add [environment variables](https://nextjs.org/docs/basic-features/environment-variables#environment-variables-on-vercel) to Vercel or modify `.env` file. We recommend to use environment variables in Vercel:
 	* `PUBLIC_URL` - your Vercel deployment URL.
-	* `SITECORE_API_KEY` - your Sitecore API key. For Sitecore XM, the `SITECORE_API_KEY` will be your Sitecore SSC API key. For Sitecore Experience Edge, this will be the API key provisioned here.
+	* `SITECORE_API_KEY` - your Sitecore API key. For Sitecore XM, the `SITECORE_API_KEY` will be your Sitecore SSC API key. For Sitecore Experience Edge, this will be the [API key provisioned here](https://doc.sitecore.com/developers/101/developer-tools/en/test-your-queries.html).
 	* `SITECORE_API_HOST` - your Sitecore API hostname. The `SITECORE_API_HOST` is not required for Sitecore Experience Edge.
 	* `JSS_EDITING_SECRET` - your secret token. The `JSS_EDITING_SECRET` is optional for deployments but necessary if you want to use the Experience Editor with your Next.js Vercel deployment. Read about [connecting your Next.js application to the Experience Editor](/docs/nextjs/experience-editor/walkthrough).
 	* `GRAPH_QL_ENDPOINT` - your GraphQL endpoint. The `GRAPH_QL_ENDPOINT` is required for Sitecore Experience Edge. For Sitecore XM, this is typically optional. By default, the endpoint is calculated using the resolved Sitecore API hostname + the `graphQLEndpointPath` defined in your `package.json`.
 3. Push the changes to your Git provider.
 
+## Video demonstration
+
+<p>
+  Watch the following video to see the deployment process and get some tips.
+  <iframe width="672" height="378" src="https://www.youtube.com/embed/s_F5hoNsvOo" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
+</p>
+
 ## `publish:end` webhook invocation
 
 Using the `publish:end` webhook, you can automatically trigger static site generation, cache clearing, and other automated activities with webhook support.
```

Comment:
With the new section for Edge webhooks, does this section need to be more clear that this is specific to Sitecore XM? Maybe change the heading to `Webhooks on Sitecore XM` or add something to call that out in the first sentence.